packages/app-store/slackmessaging/api/interactiveHandler.ts

```diff
@@ -1,13 +1,15 @@
 import { NextApiRequest, NextApiResponse } from "next";
 
 import createEvent from "../lib/actions/createEvent";
+import slackVerify from "../lib/slackVerify";
 
 enum InteractionEvents {
   CREATE_EVENT = "cal.event.create",
 }
 
 export default async function interactiveHandler(req: NextApiRequest, res: NextApiResponse) {
   if (req.method === "POST") {
+    slackVerify(req, res);
```

Comment:
Should this be `await slackVerify(...`?

Reply:
You're right! Will make the change idk how I missed that